packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -147,22 +143,14 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
 
     /// @notice Getter for the Guardian
     /// @custom:legacy
```

Comment:
The lowercase getter is not legacy

Reply:
We will want to keep the legacy `GUARDIAN()(address)` getter around unfortunately

Reply:
OK, I can revert that commit, but are you sure? I was addressing [this comment](https://github.com/ethereum-optimism/optimism/pull/7310#discussion_r1331666133).